extension/src/NABfunctions.ts

```diff
@@ -288,4 +283,26 @@ function showErrorAndLog(error: Error) {
     vscode.window.showErrorMessage(error.message);
     console.log(`Error: ${error.message}`);
     console.log(`Stack trace: ${error.stack}`);
+}
+
+export async function matchTranslations() {
+    console.log('Running: MatchTranslations');
+    let replaceSelfClosingXlfTags =  Settings.getConfigSettings()[Setting.ReplaceSelfClosingXlfTags];
+    let formatXml = true;
+    try {
+        let langXlfFiles = await WorkspaceFunctions.getLangXlfFiles();
+        console.log('Matching translations for:', langXlfFiles.toString());
+        langXlfFiles.forEach( xlfUri => {
+            let xlfDoc = Xliff.fromFileSync(xlfUri.fsPath, 'UTF8');
+            let matchResult = LanguageFunctions.matchTranslations(xlfDoc);
+            if (matchResult > 0) {
+                xlfDoc.toFileSync(xlfUri.fsPath, replaceSelfClosingXlfTags, formatXml, 'UTF8');
+            }
+            vscode.window.showInformationMessage(`Found ${matchResult} matches in ${xlfUri.path.replace(/^.*[\\\/]/, '')}.`);
```

Comment:
do we really want this message inside the loop?

Reply:
I belive they would stack so it would be pretty neat... up to a certain number of files of course.

Reply:
ok, then. We'll change if someone complains :)